packages/x-charts/src/Gauge/Gauge.tsx

```diff
@@ -0,0 +1,150 @@
+import PropTypes from 'prop-types';
+import composeClasses from '@mui/utils/composeClasses';
+import { GaugeContainer, GaugeContainerProps } from './GaugeContainer';
+import { GaugeValueArc } from './GaugeValueArc';
+import { GaugeReferenceArc } from './GaugeReferenceArc';
+import { GaugeClasses, getGaugeUtilityClass } from './gaugeClasses';
+import { GaugeValueText, GaugeValueTextProps } from './GaugeValueText';
+
+export interface GaugeProps extends GaugeContainerProps, Pick<GaugeValueTextProps, 'text'> {
+  classes?: Partial<GaugeClasses>;
+  children?: React.ReactNode;
+}
+
+const useUtilityClasses = (props: GaugeProps) => {
+  const { classes } = props;
+
+  const slots = {
+    root: ['root'],
+    valueArc: ['valueArc'],
+    referenceArc: ['referenceArc'],
+    valueText: ['valueText'],
+  };
+
+  return composeClasses(slots, getGaugeUtilityClass, classes);
+};
+
+function Gauge(props: GaugeProps) {
+  const { text, children, classes: propsClasses, ...other } = props;
+  const classes = useUtilityClasses(props);
+  return (
+    <GaugeContainer {...other} className={classes.root}>
+      <GaugeReferenceArc className={classes.referenceArc} />
+      <GaugeValueArc className={classes.valueArc} />
+      <GaugeValueText className={classes.valueText} text={text} />
+      {children}
+    </GaugeContainer>
+  );
+}
+
+Gauge.propTypes = {
+  // ----------------------------- Warning --------------------------------
+  // | These PropTypes are generated from the TypeScript type definitions |
+  // | To update them edit the TypeScript types and run "yarn proptypes"  |
+  // ----------------------------------------------------------------------
+  children: PropTypes.node,
+  classes: PropTypes.object,
+  className: PropTypes.string,
+  /**
+   * The radius applied to arc corners (similar to border radius).
+   * Set it to '50%' to get rounded arc.
+   * @default 0
+   */
+  cornerRadius: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  /**
+   * The x coordinate of the arc center.
+   * Can be a number (in px) or a string with a percentage such as '50%'.
+   * The '100%' is the width the drawing area.
+   */
+  cx: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  /**
+   * The y coordinate of the arc center.
+   * Can be a number (in px) or a string with a percentage such as '50%'.
+   * The '100%' is the height the drawing area.
+   */
+  cy: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
+  desc: PropTypes.string,
+  /**
+   * If `true`, the charts will not listen to the mouse move event.
+   * It might break interactive features, but will improve performance.
+   * @default false
+   */
+  disableAxisListener: PropTypes.bool,
+  /**
+   * The end angle (deg).
+   * @default 360
+   */
+  endAngle: PropTypes.number,
+  /**
+   * The height of the chart in px. If not defined, it takes the height of the parent element.
+   * @default undefined
+   */
+  height: PropTypes.number,
```

Comment:
I'm curious, why not?

```suggestion
  /**
   * The height of the chart in px. If not defined, it takes the height of the parent element.
   */
  height: PropTypes.number,
```

It looks funny on the API page:

<img width="761" alt="SCR-20240215-oysy" src="https://github.com/mui/mui-x/assets/3165635/523ec24e-bee5-4be4-9328-a9151936c5bb">